docs/api.md

```diff
@@ -225,9 +260,63 @@ curl -X POST http://localhost:11434/api/pull -d '{
 }
 ```
 
-## Generate Embeddings
+## Push a Model
 
+```shell
+POST /api/pull
 ```
+
+Upload a model to a model registry. Requires your public key to be added to the registry first.
```

Comment:
```suggestion
Upload a model to a model registry. Requires registering for ollama.ai and adding a public key first.
```
I suggest making this ollama.ai specific because we don't have any documentation around hosting your own model registry yet.